src/man/sssd.conf.5.xml

```diff
@@ -542,6 +542,23 @@
                             </para>
                         </listitem>
                     </varlistentry>
+                    <varlistentry>
+                        <term>lookup_order</term>
+                        <listitem>
+                            <para>
+                                Comma separated list of domains and subdomains
+                                representing the lookup order that will be
+                                followed.
+                                The list doesn't have to include all possible
+                                domains as the missing domains will be looked up
+                                based on the order they're presented in the
+                                <quote>domains</quote> configuration option.
```

Comment:
Maybe you should say as well that sub-domains which are not listed are looked up in a random order for ewach parent domain.